README.md

```diff
@@ -1 +1,60 @@
-# ul-image-api
\ No newline at end of file
+# UL Image API
```

Comment:
Shouldn't this be partnered with a GPII organisation repository as with all the others?